src/Bridge/Symfony/Routing/RouteNameResolver.php

```diff
@@ -40,7 +42,7 @@ public function getRouteName(string $resourceClass, bool $collection): string
             $operation = $route->getDefault(sprintf('_api_%s_operation_name', $operationType));
             $methods = $route->getMethods();
 
-            if ($resourceClass === $currentResourceClass && null !== $operation && (empty($methods) || in_array('GET', $methods))) {
+            if ($resourceClass === $currentResourceClass && null !== $operation && (empty($methods) || in_array('GET', $methods, true))) {
```

Comment:
I really don't like such implicit conversions... And I don't like `empty` too...

My preference would be `0 === count($methods)`, but I understand there is a  performance overhead (how much?) for the function call.